tests/rhui3_tests/test_client_management.py

```diff
@@ -56,7 +93,56 @@ def test_05_create_cli_rpm():
     Expect.enter(connection, 'q')
     Expect.ping_pong(connection, "test -f /root/test_cli_rpm-3.0/build/RPMS/noarch/test_cli_rpm-3.0-1.noarch.rpm && echo SUCCESS", "[^ ]SUCCESS")
 
-def test_06_create_docker_cli_rpm():
+def test_08_remove_rhui_conf_rpm():
+    '''
+       remove rhui configuration rpm from client
+    '''
+    Util.remove_conf_rpm(cli)
```

Comment:
Exactly, to avoid any conflict with a newly installed cli configuration rpm. 

ok, what naming do you suggest for that method?